dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -617,6 +617,23 @@
         # This setting can also be overridden using the environment variable ZEEBE_BROKER_CLUSTER_MEMBERSHIP_SYNCINTERVAL
         # syncInterval: 10s
 
+      # Configure the parameters used to propagate the dynamic cluster configuration across brokers and gateways
+      # dynamicConfig:
+        # Configure the parameters used for gossiping the configuration
+        # gossip:
```

Comment:
:wrench: May be we could add a note to clarify that this is not the same as membership gossip?

Reply:
Ok I'll modify it to: 
` # Configure the parameters used for gossiping the dynamic cluster configuration. Note that this is not related to cluster membership gossip`